tutorials/notebook_test_util.py

```diff
@@ -0,0 +1,119 @@
+import nbformat
+import os
+import shutil
+import sys
+import argparse 
+
```

Comment:
so to make it easier to maintain this file, I think it should receive a description of what it is used for.